Generator/BotBuilderVSIX-V4/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("BotBuilderVSIX")]
+[assembly: AssemblyDescription("")]
```

Comment:
Should these attributes be filled out?

Reply:
Yes. I will fill them out.

Reply:
@FranciscoPonceGomez - Any luck wrapping this up? We should get it checked in soon.

Reply:
Working on it. I will get this done before noon.